ome_zarr/reader.py

```diff
@@ -57,7 +57,7 @@ def __init__(
             self.specs.append(PlateLabels(self))
         elif Plate.matches(zarr):
             self.specs.append(Plate(self))
-            self.add(zarr, plate_labels=True)
+            # self.add(zarr, plate_labels=True)
```

Comment:
maybe a comment indicating why it is commented out?

Reply:
Agreed. @joshmoore @will-moore should https://github.com/ome/ome-zarr-py/issues/65 act as the reference issue for re-enabling/rewriting `PlateLabels`? In which case the code could directly link to it

Reply:
Yes, I think so - Don't fully know the reason for each failure, but they seem closely related

Reply:
I'll leave you to open a follow-up PR. Merging for a pre-release